docs/user_guide.md

```diff
@@ -940,6 +967,15 @@ root account arguments:
                         not require authentication otherwise.
 ~~~~~~~~~~~~~~~~~~~~~
 
+### Enfore secure socket (SSL)
```

Comment:
Perhaps we should link to some generally available guide which explains how to make an SSL certificate?